app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/TagsManager.java

```diff
@@ -100,130 +105,112 @@ boolean clearTag(Tag tag) {
-            TAG_DATETIME_DIGITIZED,
-            TAG_DATETIME_ORIGINAL,
-            TAG_DEVICE_SETTING_DESCRIPTION,
-            TAG_EXIF_VERSION,
-            TAG_FILE_SOURCE,
-            TAG_FLASHPIX_VERSION,
-            TAG_SPATIAL_FREQUENCY_RESPONSE,
-            TAG_SPECTRAL_SENSITIVITY,
-            TAG_SUBSEC_TIME,
-            TAG_SCENE_TYPE,
-            TAG_RELATED_SOUND_FILE,
-            TAG_IMAGE_UNIQUE_ID,
-            TAG_MAKER_NOTE,
-            TAG_OECF,
-            TAG_SUBSEC_TIME_DIGITIZED,
-            TAG_SUBSEC_TIME_ORIGINAL,
-            TAG_GPS_AREA_INFORMATION,
-            TAG_GPS_DEST_DISTANCE_REF,
-            TAG_GPS_DEST_LATITUDE_REF,
-            TAG_GPS_DEST_LONGITUDE_REF,
-            TAG_GPS_MAP_DATUM,
-            TAG_GPS_MEASURE_MODE,
-            TAG_GPS_SATELLITES,
-            TAG_GPS_SPEED_REF,
-            TAG_GPS_STATUS,
-            TAG_USER_COMMENT,
-            TAG_GPS_TRACK_REF,
-            TAG_GPS_VERSION_ID,
-            TAG_INTEROPERABILITY_INDEX);
-
-
+    /**
+     * We are using these tags below N android version.
+     * Annotation prevents from generating lint error.
+     */
     @RequiresApi(api = Build.VERSION_CODES.N)
-    private static final List<String> INTEGER_TAGS_LIST_API_24 = Arrays.asList(
-            TAG_PHOTOMETRIC_INTERPRETATION,
-            TAG_BITS_PER_SAMPLE,
-            TAG_COMPRESSION,
-            TAG_JPEG_INTERCHANGE_FORMAT,
-            TAG_JPEG_INTERCHANGE_FORMAT_LENGTH,
-            TAG_PLANAR_CONFIGURATION,
-            TAG_RESOLUTION_UNIT,
-            TAG_ROWS_PER_STRIP,
-            TAG_SAMPLES_PER_PIXEL,
-            TAG_STRIP_BYTE_COUNTS,
-            TAG_STRIP_OFFSETS,
-            TAG_TRANSFER_FUNCTION,
-            TAG_Y_CB_CR_POSITIONING,
-            TAG_Y_CB_CR_SUB_SAMPLING,
-            TAG_COLOR_SPACE,
-            TAG_CONTRAST,
-            TAG_CUSTOM_RENDERED,
-            TAG_EXPOSURE_MODE,
-            TAG_EXPOSURE_PROGRAM,
-            TAG_FOCAL_LENGTH_IN_35MM_FILM,
-            TAG_FOCAL_PLANE_RESOLUTION_UNIT,
-            TAG_GAIN_CONTROL,
-            TAG_ISO_SPEED_RATINGS,
-            TAG_LIGHT_SOURCE,
-            TAG_METERING_MODE,
-            TAG_PIXEL_X_DIMENSION,
-            TAG_PIXEL_Y_DIMENSION,
-            TAG_SATURATION,
-            TAG_SCENE_CAPTURE_TYPE,
-            TAG_SENSING_METHOD,
-            TAG_SHARPNESS,
-            TAG_SUBJECT_AREA,
-            TAG_SUBJECT_DISTANCE_RANGE,
-            TAG_SUBJECT_LOCATION,
-            TAG_GPS_DIFFERENTIAL,
-            TAG_THUMBNAIL_IMAGE_LENGTH,
-            TAG_THUMBNAIL_IMAGE_WIDTH);
-
-
-    @RequiresApi(api = Build.VERSION_CODES.N)
-    private static final List<String> DOUBLE_TAGS_LIST_API_24 = Arrays.asList(
-            TAG_DIGITAL_ZOOM_RATIO,
-            TAG_EXPOSURE_BIAS_VALUE,
-            TAG_F_NUMBER,
-            TAG_SUBJECT_DISTANCE);
+    private static final Map<String, InputType> TAG_LIST_N = new HashMap<String, InputType>() {
```

Comment:
Map you made is not a constant one.
This is a proper implementation:
  public static final Map<String, String> TAG_LIST_N;
    static {
        Map<String, InputType> map = new HashMap<String, InputType>();
        map.put("key", value);
        ...
        TAG_LIST = Collections.unmodifiableMap(map);
    }